src/envo/envo-edit.owl

```diff
@@ -44974,6 +44976,27 @@ AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#has
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03605005> "epixylon"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03605005> <http://purl.obolibrary.org/obo/ENVO_03605000>)
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_03605006> (freshwater stream)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03605006> "A stream where fresh water flows.")
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#created_by> <http://purl.obolibrary.org/obo/ENVO_03605006> <ORCID:0000-0001-9076-6066>)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#creation_date> <http://purl.obolibrary.org/obo/ENVO_03605006> "2024-01-10T19:07:00Z")
```

Comment:
Please follow the specifications here (using DC for creation date): https://github.com/EnvironmentOntology/envo/wiki/Adding-classes-to-ENVO